VMWare/vmware-esxi/tests/AUTH_CONNECTION_disconnected.json

```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "Disconnected from user root 1.2.3.4 port 33398"
+  },
+  "expected": {
+    "message": "Disconnected from user root 1.2.3.4 port 33398",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+        "authentication"
+      ],
+      "type": [
+        "info"
```

Comment:
For this event, a more suitable `event.type` will be `["end"]` as it represent a disconnection.